spec/components/rate_limiter_spec.rb

```diff
@@ -38,6 +38,48 @@
       RateLimiter.disable
     end
 
+    context 'aggressive rate limiter' do
+
+      it 'can operate correctly and totally stop limiting' do
+
+        freeze_time
+
+        # 2 requests every 30 seconds
```

Comment:
Minor but I usually use keyword arguments so that the code itself reflect what is being done instead of requiring comments like this.

Reply:
yeah the interface was defined so long ago, changing rate limiter now is going to be a massive change especially for all plugins.